resources/templates/Menu.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+         <link rel="stylesheet" href="menu.css">
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+        <script src="Resources/script.js" type="text/javascript"></script> 
+        <script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"></script>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+
+
+</head>
+<body>
+
+	<h4 style="margin-left: 45% ; margin-top: 5% ; font-size: 50px">Menú</h4>
+	<table style="width: 50% ; margin-left: 25%; margin-top: 5%; font-size: 30px " class="table table-striped table-dark">
+		
+		<tbody>
+			<tr>
+				
+				<th scope="row"><i class="fas fa-arrow-circle-right"></i></th>
+				<td>Ventas</td>
+			</tr>
+			<tr>
+				<th scope="row"><i class="fas fa-arrow-circle-right"></i></th>
+				<td>Alivios</td>
+			</tr>
+			<tr>
+				<th scope="row"><i class="fas fa-arrow-circle-right"></i></th>
+				<td>Clave</td>
+			</tr>
+			<tr>
+				<th scope="row"><i class="fas fa-arrow-circle-right"></i></th>
+				<td>Reembolso</td>
+			</tr>
+			<tr>
+				<th scope="row"><i class="fas fa-arrow-circle-right"></i></th>
+				<td>Salir</td>
+			</tr>
+		</tbody>	
+		
```

Comment:
Otra vez, todo este espacio es inútil y complica la lectura. Un poco más arriba, al definir la fila de las ventas, también hay espacio que no correspondería, y lo mismo en el tag head.